### PR TITLE
Issue/#156  code contact us page

### LINKED ISSUE #156: Code Contact Us Page
https://www.notion.so/7f3a05404d0a4ea0abf55177dcac1eef?v=fb187aa25ac44a4ebac907b2ec635acd&p=5cf0566a36e4439ab44e17a123c10aa2

### PR DESCRIPTION
**Pull request by MaksymTaratuta55**
closes #156

### CODE REVIEW

**Comment by MaksymTaratuta55** [comment]
@thecodeclimber, I've finished except "Sync into Active Campaign/Pipe Drive"

File: src/assets/svg/Icon__success.svg [hunk 1]
```diff
@@ -0,0 +1,4 @@
+<svg width="120" height="120" viewBox="0 0 120 120" fill="none" xmlns="http://www.w3.org/2000/svg">
```
Comment (thecodeclimber) [inline src/assets/svg/Icon__success.svg:1]: please have all svg file names be in lower case.  I.e., `icon__success.svg` not `Icon__success.svg`.  Thanks.

**Comment by thecodeclimber** [comment]
@MaksymTaratuta55 

We are getting the following error on Vercel:

```

17:37:54.290 | Failed to compile.
-- | --
17:37:54.290 | ModuleNotFoundError: Module not found: Error: Can't resolve '../../assets/svg/icon__success.svg' in '/vercel/workpath0/src/components/elements'
17:37:54.290 | > Build error occurred
17:37:54.291 | Error: > Build failed because of webpack errors
17:37:54.292 | at /vercel/workpath0/node_modules/next/dist/build/index.js:15:918
17:37:54.292 | at runMicrotasks (<anonymous>)
17:37:54.292 | at processTicksAndRejections (internal/process/task_queues.js:97:5)
17:37:54.292 | at async /vercel/workpath0/node_modules/next/dist/build/tracer.js:1:1441
17:37:54.320 | error Command failed with exit code 1.
17:37:54.320 | info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
17:37:54.332 | Error: Command "yarn run build" exited with 1
```

**Comment by MaksymTaratuta55** [comment]
@thecodeclimber, I've fixed vercel error

**Pull request merged**